step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesHandler.java

```diff
@@ -34,7 +35,7 @@
 
 public class TimeSeriesHandler {
 
-	private static final int FIELDS_SAMPLING_LIMIT = 1_000;
+    private static final int FIELDS_SAMPLING_LIMIT = 1_000;
```

Comment:
FIELDS_SAMPLING_LIMIT should be used as default value, but we should be able to set (optionally) a value from step.properties